packages/twenty-website/src/app/_server-utils/fetch-stargazers-count.ts

```diff
@@ -0,0 +1,26 @@
+export async function fetchStargazerCount(): Promise<string | null> {
+  /**
+   * Fetch the stargazers count from the GitHub API and revaildate every 1 hour
+   * to avoid hitting the rate limit
+   */
+  try {
+    const res = await fetch('https://api.github.com/repos/twentyhq/twenty', {
+      next: { revalidate: 3600 },
```

Comment:
Didn't know about this revalidate param, nice. We already had a function/cron to sync Github data to the Postgres db so we went this path - but yours is elegant and straightforward, I like it

Reply:
Yeah it saves on having to store simple data like this in a database unnecessarily by just storing it in memory, see reference docs here - https://nextjs.org/docs/app/building-your-application/data-fetching/fetching-caching-and-revalidating#fetching-data-on-the-server-with-fetch